paradise_papers_django/search/templates/search/index.html

```diff
@@ -1,25 +1,39 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <div class="tab">
+<div class="search_box big">
```

Comment:
Indentation make your code readable. Seems you need to do some work with indentation here.